modules/ROOT/pages/platform/api/authentication.adoc

```diff
@@ -160,5 +160,19 @@ print(response.json())
 
 === Token expiration
 
-If you send a request to the Aura API while authenticated with an expired access token, you will receive a 403 Forbidden response.
-You will need to obtain a new token to continue using the API.
+Tokens are temporary access tokens used for authentication when making API requests. 
```

Comment:
This should be moved to where the tokens are first introduced (I think in the "Authenticating API requests" section).